package.json

```diff
@@ -42,20 +42,20 @@
     }
   },
   "dependencies": {
-    "js-beautify": "^1.8.9",
-    "safer-eval": "^1.3.0"
+    "js-beautify": "^1.10.0",
+    "safer-eval": "^1.3.2"
```

Comment:
I think it should be moved to `devDependencies`

Reply:
Thanks for the notice